src/ui/components/Passport/Passport.module.css

```diff
@@ -68,7 +68,7 @@
 .videoExample {
   border-bottom-left-radius: 6px;
   border-bottom-right-radius: 6px;
-  z-index: 100;
+  z-index: 5;
```

Comment:
Don't we have z-index CSS variables? (I think you added them) Why don't we use them?

Reply:
I don't recall doing that, but can look later

Reply:
Moved to css vars